aptos/token_bridge/sources/utils.move

```diff
@@ -19,28 +19,35 @@ module token_bridge::utils {
         ret
     }
 
-    //TODO - finish and test normalized and denormalize functions
-
-    // public entry fun normalize_amount(amount: U256, decimals: u8): U256 {
-    //     if (decimals > 8) {
-    //         amount = u256::div(amount, 10 ** (decimals - 8));
-    //     };
-    //     amount
-    // }
+    public entry fun normalize_amount(amount: U256, decimals: u8): U256 {
+         if (decimals > 8) {
+            let n = decimals - 8;
+            while (n > 0){
```

Comment:
not sure if we need a loop here, just
```
amount = u256::div(amount, from_u64(10^n));
```

Reply:
tried this and it doesn't work, I don't think ^ is exponentiation, couldn't figure out the syntax for it or find it...

Reply:
ah looks like `^` is exclusive or